openrct2.music.alternative.json

```diff
@@ -37,6 +37,12 @@
             "samples": [
                 "replacements/openrct2.music.prehistoric/music/0.flac"
             ]
+        },
+        {
+            "id": "rct2.music.ragtime",
+            "samples": [
+                "replacements/openrct2.music.ragtime/music/0.flac"
+            ]
```

Comment:
This needs to go in `openrct2.music.cover.json`

it would probably be best to have cover music in it's own folder. Might need to restructure. Rename replacements to alternative and add cover. Cover pieces won't become objects, just an assetpack.